docs/INTEGRATION_FOREGROUND_FILTERS.md

```diff
@@ -0,0 +1,123 @@
+## Foreground Filters
+Foreground filters provide a way for pre-applying a filter on page load. The applied filter will be applied to the URL and can be removed as any other applied filter would. Foreground filtering is accomplished by setting the inital UrlManager state; this can be used for setting various states, but only filtering will be covered in this document.
+
+Foreground filters are only usable with a SearchController.
+
+In the simplified example below, a foreground filter is used to pre-apply a filter for the `on_sale` field. 
+
+```typescript
+const config = {
+	client: {
+		globals: {
+			siteId: 'abc123',
+		},
+	},
+	controllers: {
+		search: [
+			{
+				url: {
+					initial: {
+						parameters: {
+							filter: {
+								state: {
+									on_sale: ['yes'],
+								}
+							},
+						},
+					}
+				},
+				config: {
+					id: 'search',
+				},
+			},
+		],
+	},
+};
+
+const snap = new Snap(config);
+```
+
+
+The `initial.parameters` object takes `UrlManager` state parameters, such as filter, sort, page, and pageSize. This feature is likely to mostly be used used with filter, but any parameters including custom ones could be utilized. 
```

Comment:
Duplicate word - "used used" here: "This feature is likely to mostly be used used with filter,"
Also consider rephrasing to:
"This feature could be utilized by any parameter however is most likely to be used with the filter parameter"